classification/apply_HLDA.m

```diff
@@ -0,0 +1,61 @@
+function out = apply_HLDA(epo, ival, C)

```

Comment:
We could get rid of the ival altogether (both in the train and apply functions), and simply pass the final X (with dimensions (time * channels) X samples) and the labels Y (similar to train_LDA in the toolbox). In this case, the needed interval should already be selected before passing X to the training function. In this way, the arguments and the  epo.t problems should be solved.